### PR TITLE
Bootstrap functionality

### PR DESCRIPTION
**Pull request by elaineshyu**

### CODE REVIEW

**Review by dbehailu — APPROVED** [review]

File: src/main/webapp/user-page.jsp [hunk 1]
```diff
@@ -92,7 +92,7 @@ String uploadUrl = blobstoreService.createUploadUrl("/my-form-handler"); %>
 
   <!-- <form id="message-form" action="/messages" method="POST" class="hidden"> -->
 
-    <form id="message-form" action="<%= uploadUrl %>" method="POST" enctype="multipart/form-data">
+    <form id="message-form" action="<%= uploadUrl %>" method="POST" enctype="multipart/form-data" class="hidden">
```
Comment (dbehailu) [inline src/main/webapp/user-page.jsp:95]: what does class="hidden" do again?

**Review by dbehailu — COMMENTED** [review]
.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -22,6 +22,107 @@
     <script src="/js/navigation-loader.js"></script>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <script>
+      //code credit to w3schools
+      function autocomplete(inp, arr) {
```
Comment (dbehailu) [inline src/main/webapp/index.html:26]: This is such a cool feature!! Nice 👍 !

File: src/main/webapp/user-page.jsp (hunk 1 above)
Comment (elaineshyu) [reply src/main/webapp/user-page.jsp:95]: it's for when the user page to see if it's your page; if it is your page, it'll show the input / message boxes, but if it isn't, it won't show them :)

Comment (dbehailu) [reply src/main/webapp/user-page.jsp:95]: ohhhh nice! good call 👍

**Pull request merged**